app_zarzadzanie_budzetem/app_zarz_bud_MENU.py

```diff
@@ -0,0 +1,36 @@
+def menu():
+    #lista_akcji = ListaZakupow()
+
+    while True:
+        print("\nMenu: \n"
+              "1. Dodanie przychodów do budżetu\n"
+              "2. Odejmowanie wydatków od budżetu\n"
+              "3. Dodanie stałych/cyklicznych wydatków\n"
+              "4. Export raportu di pliku\n"
+              "5. Przeliczenie Walut\n"
+              "6. Historia Wydatków\n"
+              "0. Wyjście z Aplikacji")
+        wybor = input("Wybierz opcję: ")
+
+
+        if wybor == "1":
```

Comment:
Proponuję zmianę na angielski, np.
```suggestion
        if choice == "1":
```